composer.json

```
@@ -30,6 +30,7 @@
         "php": "^7.4 || ^8",
         "composer/installers": "^1.9",
         "cweagans/composer-patches": "^1.7",
+        "discoverygarden/dgi_image_discovery": "*",
```

Probably shouldn't leave this wide open:

```suggestion
        "discoverygarden/dgi_image_discovery": "^1",
```